src/modules/PreviewTalentModule/index.tsx

```
@@ -0,0 +1,106 @@
+import { useEffect, useState } from 'react';
+import { Button } from '@/components';
+import Experience, { ExperienceDetail } from '@/components/ui/experience';
+import UserProfileCard, { UserProfile } from '@/components/ui/profile';
+import { ArrowLeft, BookmarkIcon } from 'lucide-react';
+import Location from '@/components/ui/location';
+import Cookies from 'js-cookie';
```

ESLint

[error] 'Cookies' is defined but never used.

(@typescript-eslint/no-unused-vars)